### PR TITLE
Fix/api/bop 95

### PR DESCRIPTION
**Pull request by ian-bpoc**
# Pull Request Review Comments

## Overview
- **Purpose of the PR**:
    - [ /] Bug fix
    - [/ ] Refactoring

## Specific Areas for Verification
- Please specify any particular requests or questions for the reviewer here.

## Suggestions/Concerns
- If you have any suggestions or concerns, please mention them here.

---

_**Additional Comments for the Reviewer (Optional)**_

- Add any additional comments or information for the reviewer here.

### CODE REVIEW

**Review by edward-stalley-work — CHANGES_REQUESTED** [review]
Apologies for the delay Ian.
I have had a brief look at the PR and noticed that the naming conventions in the API/model sheet are not always being followed. I was away when the initial API, URLS updates PR were done so I never got to check it properly (2 weeks ago).

So Before I look more in depth, could you take a look at the design sheet and make sure all urls, serializers, class names follow same convention!

You can remove the 'View' part.
In the design document I have as an example

Screenshot 2024-10-02 at 15 35 26

You can change 'UserListView' to 'UserList' .
Everything else please follow and keep an eye on where plural is and is not used.

**Comment by edward-stalley-work** [comment]
Also please make sure to test all pages and functions (Create, Update , Delete) to make sure they are not broken with any of the changes/renaming.

**Comment by ian-bpoc** [comment]
> Also please make sure to test all pages and functions (Create, Update , Delete) to make sure they are not broken with any of the changes/renaming.

The moment serializer and views naming is changed it is okay Ed, unless the url is modified thats when i check and test the data

File: backend/api/views.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 from .serializers import (
     AllPlanningSerializer,
     CostOfSalesSerializer,
-    CreateProjectsSerializers,
+    ProjectsCreateSerializer,
     CreateResultsSerializers,
```
[the remark this answers is not in the document]
Comment (ian-bpoc) [reply backend/api/views.py:13]: Will implement it right away

**Comment by ian-bpoc** [comment]
> Apologies for the delay Ian. I have had a brief look at the PR and noticed that the naming conventions in the API/model sheet are not always being followed. I was away when the initial API, URLS updates PR were done so I never got to check it properly (2 weeks ago).
> 
> So Before I look more in depth, could you take a look at the design sheet and make sure all urls, serializers, class names follow same convention!
> 
> You can remove the 'View' part. In the design document I have as an example
> 
> Screenshot 2024-10-02 at 15 35 26
> You can change 'UserListView' to 'UserList' . Everything else please follow and keep an eye on where plural is and is not used.

in my current pr UserList already exist and not UserListView

**Pull request closed without merging**